fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -161,6 +161,7 @@ ROLLUP: 'ROLLUP';
 ROW: 'ROW';
 ROWS: 'ROWS';
 ROW_NUMBER: 'ROW_NUMBER';
+SUBMIT: 'SUBMIT';
```

Comment:
should be a unReserved too to keep compatible if there is a column named `submit`